### PR TITLE
Ca feature/template endpoint

### PR DESCRIPTION
**Pull request by cagodoy**
🐒  TEMPLATE CRUD ENDPOINT

```
{
  emitDate: {
    type: Date,
    required: true,
  },
  emisor: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
    required: true,
  },
  receptionDate: {
    type: Date,
  },
  receptor: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
  },
  auxiliary: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
    required: true,
  },
  driver: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
    required: true,
  },
  Bus: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Bus',
    required: true,
  },
  passes: {
    type: [mongoose.Schema.Types.ObjectId],
    ref: 'Pass',
  },
  // TODO: add folio attr with autoincrement
  initialRabbit: [{
    deck: {
      type: [mongoose.Schema.Types.ObjectId],
      ref: 'Deck',
      required: true,
    },
    correlative: {
      type: Number,
      required: true,
    },
  }],
  isClosed: {
    type: Boolean,
    require: true,
    default: false,
  }
}
```

### CODE REVIEW

**Comment by mtavano** [comment]
🚀 🚀

File: src/app/api/template/template.model.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    },
+  }],
+  isClosed: {
+    type: Boolean,
+    require: true,
+    default: false,
+  }
+})
+
+export default mongoose.model('Pike', templateSchema)
```
Comment (eseceve) [inline src/app/api/template/template.model.ts]: export default mongoose.model('Template', templateSchema)

Comment (mtavano) [inline src/app/api/template/template.model.ts]: sassy copy paste! 👀

**Comment by cagodoy** [comment]
go

**Pull request merged**